www/webapp/src/components/ActivateAccountActionHandler.vue

```diff
@@ -3,16 +3,33 @@
     <div class="text-center" v-if="captcha_required && !success">
         <generic-captcha
             @update="(id, solution) => {setCaptchaPayload(id, solution)}"
-            tabindex="3"
+            tabindex="1"
             ref="captchaField"
         />
+        <v-layout class="justify-center">
+          <v-checkbox
+                v-model="terms"
```

Comment:
It's interesting that we don't enforce terms agreement at the server side (both here and at the 'regular' signup form).

Reply:
If I recall correctly, that's because the API is programmatic and can't enforce the terms text to be displayed next to the terms payload field boolean value.